include/overlay006/battle_params.h

```diff
@@ -31,7 +31,7 @@ typedef struct {
     UnkStruct_0207D99C * unk_E4;
     UnkStruct_02026324 * unk_E8;
     PCBoxes * unk_EC;
-    UnkStruct_0202CC84 * unk_F0[4];
+    ChatotCryData * unk_F0[4];
```

Comment:
```suggestion
    ChatotCryData *chatotCry[MAX_BATTLERS];
```

Reply:
by renaming this though to chatotCry I believe we would have to rename every time something points to unk_F0 in this struct, and since the struct can be initiated with many names I'm not sure how to go about finding them all in a humane way

Reply:
Fairy, I typically just treat it as necessary grunt work, but we can leave this until someone gets around to documenting the struct in its entirety.